docs/reference/mount.md

```diff
@@ -21,13 +21,18 @@ This functionality is currently in limited support and is a Read-Only file syste
 ⚠️ No installation is required. Please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access to the Everest binary.
 {: .note }
 
+****
+Everest mount supports writing to the file system and now available for NFS (the default protocol for MacOS)!
```

Comment:
Nit; I don't like `now available` comments since it's never maintained. Maybe just
```suggestion
Everest mount supports writing to the file system for NFS (the default protocol for MacOS)!
```

Reply:
I wanted to emphasize that it's in experimental mode.